sencha-workspace/packages/slate-cbl/src/view/demonstrations/SkillList.js

```diff
@@ -116,16 +128,112 @@ Ext.define('Slate.cbl.view.demonstrations.SkillList', {
+
+                            '<hr class="skill-list-detail-separator">',
+
+                            `<div class="skill-list-detail-columns">
+                                <tpl if="this.linksExist(Demonstration)">
+                                    <div class="skill-list-detail-col -links">
+                                        <h4 class="skill-list-detail-heading">Links</h4>
+
+                                        <tpl if="Demonstration.ArtifactURL">
+                                            <div class="skill-list-detail-group">
+                                                <h5 class="skill-list-detail-subheading">Artifacts</h5>
+                                                <ul class="skill-list-links">
+                                                    <li class="skill-list-link-item">
+                                                        <a href="{Demonstration.ArtifactURL}" class="skill-list-link">
+                                                            <i class="fa fa-link skill-list-link-icon"></i>
+                                                            <div class="skill-list-link-label">{Demonstration.ArtifactURL}</div>
+                                                        </a>
+                                                    </li>
+                                                </ul>
+                                            </div>
+                                        </tpl>
+                                        <tpl if="Demonstration.StudentTask.Task.Attachments && Demonstration.StudentTask.Task.Attachments.length">
+                                            <div class="skill-list-detail-group">
+                                                <h5 class="skill-list-detail-subheading">Task Attachments</h5>
+                                                <ul class="skill-list-links">
+                                                    <tpl for="Demonstration.StudentTask.Task.Attachments">
+                                                        <li class="skill-list-link-item">
+                                                            <a href="{URL}" target="_blank" class="skill-list-link">
+                                                                <i class="fa fa-link skill-list-link-icon"></i>
+                                                                <div class="skill-list-link-label">{[this.linkText(values)]}</div>
+                                                            </a>
+                                                        </li>
+                                                    </tpl>
+                                                </ul>
+                                          </div>
+                                        </tpl>
+                                        {% var attachments = this.getActiveAttachments(values.Demonstration.StudentTask.Attachments) %}
+                                        <tpl if={attachments.length}>
```

Comment:
I think it's a fluke here that this does anything significant, I think what you're accomplishing here is forcing XTemplate to process it as a raw javascript snippet by putting a dummy code block around it. The preferred way to force raw expression is to wrap in parens, and it should be quoted either way:

```suggestion
                                        <tpl if="(attachments.length)">
```

Reply:
I realized I was no longer using the attachments variable defined on the previous line elsewhere, so I removed the variable and updated the conditional to call a template function.  a5cc892025093044422b5ae864cc5f29b8e724d2